pipeline/generate-preview-csharp-calcvalues-package.yaml

```diff
@@ -32,7 +32,16 @@ spec:
       source /workspace/source/.jx/variables.sh
       cd ./registry
       export PREVIEW_TIMESTAMP=$(date +%s)
-      /CreateCsharpPackage.sh http://$(cat ../pod_ip)/openapi.json 0.0.$PULL_NUMBER-preview-$PREVIEW_TIMESTAMP Mqube.$SwaggerServiceName.Client $REPO_OWNER $REPO_NAME
+      
+      SWAG_PATH=$(find /workspace/source -type f -name "swagger.json")
+      if test $SWAG_PATH ; then
+        echo Swagger docs found in repository
+      else
+        SWAG_PATH=http://$(cat ../pod_ip):5000/swagger/v1/swagger.json
```

Comment:
Some python services uses kubernetes service port 80 (as here without :5000) and /openapi.json instead of /swagger.json. Need to drop :5000 here or it will break build. And also check others for openapi.json